build.gradle

```diff
@@ -68,9 +77,10 @@ tasks.idea.dependsOn(tasks.copyInspections)
 
 tasks.register('buildIgnoreRatList', Exec) {
   description 'Builds a list of ignored files for the rat task from the unversioned git files'
-  commandLine 'bash', '-c', 'git clean --force -d --dry-run -x | cut -c 14-'
+  commandLine 'bash', '-c', 'git clean --force -d --dry-run -x | grep -v "Would skip" | cut -c 14- && ' +
+          'git clean --force -d --dry-run -x | grep "Would skip" | cut -c 23-\n'
   doFirst {
-    standardOutput new FileOutputStream("${buildDir}/.rat-excludes.txt")
+    standardOutput new FileOutputStream(".rat-excludes.txt")
```

Comment:
I think it was just build troubleshooting - not sure why I ended up moving it, but I'll try putting it back and see if there was some reason I moved it.

Reply:
Interestingly just reverting this and related changes just plain broke the build, but I've found a way around it - there were some super-weird things with `buildDir` just not working (as it's not a string, it's a file, so that include or somewhere else we were using it was putting `{}`s around it in the final string, which failed).